Content.Server/Chat/SpeakOnTriggerComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Shared.Dataset;
+using Robust.Shared.Prototypes;
+using Content.Shared.DeviceLinking;
+
+namespace Content.Server.Chat;
+
+/// <summary>
+///     Makes the entity speak when triggered. If the item has UseDelay component, the system will respect that cooldown.
+/// </summary>
+[RegisterComponent]
+public sealed partial class SpeakOnTriggerComponent : Component
+{
+    /// <summary>
+    ///     The identifier for the dataset prototype containing messages to be spoken by this entity.
+    /// </summary>
+    [DataField(required: true)]
+    public ProtoId<LocalizedDatasetPrototype> Pack { get; private set; }
```

Comment:
```suggestion
    public ProtoId<LocalizedDatasetPrototype> Pack = string.Empty;
```
Oh, and I think getters and setters are no longer needed for components.